hbase-common/src/main/java/org/apache/hadoop/hbase/util/ReservoirSample.java

```diff
@@ -41,7 +41,7 @@ public class ReservoirSample<T> {
   private int n;
 
   public ReservoirSample(int k) {
-    Preconditions.checkArgument(k > 0, "negative sampling number(%d) is not allowed");
+    Preconditions.checkArgument(k > 0, "negative sampling number(%s) is not allowed", k);
```

Comment:
Checkstyle warning cleanup?

Reply:
Not only checkstyle, if there are errors the output is incorrect...
Maybe we should file an issue to apply this to all branches.

Reply:
Yes I guess so.